javascript/Gruntfile.js

```diff
@@ -0,0 +1,134 @@
+/*jslint node: true */
+"use strict";
+module.exports = function (grunt) {
+ grunt.initConfig({
+  pkg: grunt.file.readJSON('package.json'),
+
+  clean: {
+   build: ['../olad/www/index.html', '../olad/www/js', '../olad/www/css', '../olad/www/img', '../olad/www/libs', '../olad/www/views']
+  },
+
+  copy: {
+   build: {
+    files: [
+    {
+     expand: true,
+     src: 'src/img/*',
+     dest: '../olad/www/img/',
+     flatten: true,
+     filter: 'isFile'
+    },
+    {
+     expand: true,
+     src: 'src/views/*',
+     dest: '../olad/www/views/',
+     flatten: true,
+     filter: 'isFile'
+    },
+    {
+     expand: true,
+     cwd: 'src/libs',
+     src: '**',
+     dest: '../olad/www/libs',
+     flatten: false
+    },
+    {
+     expand: false,
+     src: 'src/index.html',
+     dest: '../olad/www/index.html',
+     flatten: true
+    }
+    ]
+   }
+  },
+
+  bower: {
+   dev: {
+    options: {
+     targetDir: 'src/libs',
+     install: true,
+     copy: true,
+     cleanup: true,
+     layout: 'byComponent'
+    }
+   }
+  },
+
+  uglify: {
+   default: {
+    files: {
+     '../olad/www/js/app.min.js': ['../olad/www/js/app.js']
+    },
+    options: {
+     mangle: true,
+     sourceMap: true,
+     sourceMapName: '../olad/www/js/app.min.js.map',
+     banner: "/**\n"+
```

Comment:
The license doesn't particularly need to be in the minified version. We've never bothered before, although @nomis52 may want it if we can.